modules/internal/localeModels/numa/LocaleModel.chpl

```diff
@@ -404,12 +404,12 @@ module LocaleModel {
   proc chpl_executeOn(loc: chpl_localeID_t, // target locale
                       fn: int,              // on-body function idx
                       args: c_void_ptr,     // function args
-                      args_size: int(32)    // args size
+                      args_size: size_t     // args size
```

Comment:
Eesh.  Did we miss this when this change was made before?  (Really a comment, no need to change anything.)